src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -477,93 +476,92 @@ void func_80B12460(EnSyatekiNiw* this, PlayState* play) {
                        RAD_TO_BINANG(Math_FAtan2F(player->actor.world.pos.x - this->actor.world.pos.x,
                                                   player->actor.world.pos.z - this->actor.world.pos.z)) +
                            phi_f16,
-                       5, this->unk_2C8.y, 0);
-    Math_ApproachF(&this->unk_2C8.y, 3000.0f, 1.0f, 500.0f);
-    if (this->unk_296 == 2) {
-        this->unk_256 = 10;
-        this->unk_254 = this->unk_256;
+                       5, this->posRotStep.y, 0);
+    Math_ApproachF(&this->posRotStep.y, 3000.0f, 1.0f, 500.0f);
+    if (this->archeryAnimationType == 2) {
+        this->peckTimer = this->timer1 = 10;
     }
 
-    func_80B11A94(this, play, this->unk_296);
+    EnSyatekiNiw_UpdateRotateTargets(this, play, this->archeryAnimationType);
 }
 
-void func_80B128D8(EnSyatekiNiw* this, PlayState* play) {
-    if (this->unk_25E == 1) {
+void EnSyatekiNiw_ExitArchery(EnSyatekiNiw* this, PlayState* play) {
+    if (this->movementTimer == 1) {
         gSaveContext.timerState = TIMER_STATE_OFF;
     }
 }
 
-void func_80B128F8(EnSyatekiNiw* this, PlayState* play) {
+void EnSyatekiNiw_SetupRemove(EnSyatekiNiw* this, PlayState* play) {
     s16 sp26;
     s16 sp24;
 
-    Actor_SetFocus(&this->actor, this->unk_2D4);
+    Actor_SetFocus(&this->actor, this->focusYOffset);
     Actor_GetScreenPos(play, &this->actor, &sp26, &sp24);
     if ((this->actor.projectedPos.z > 200.0f) && (this->actor.projectedPos.z < 800.0f) && (sp26 > 0) &&
         (sp26 < SCREEN_WIDTH) && (sp24 > 0) && (sp24 < SCREEN_HEIGHT)) {
         this->actor.speed = 5.0f;
-        this->unk_298 = Rand_ZeroFloat(1.99f);
-        this->unk_2D8 = Rand_CenteredFloat(8000.0f) + -10000.0f;
-        this->unk_262 = 0x1E;
-        this->unk_25E = 0x64;
-        this->actionFunc = func_80B129EC;
+        this->rotYFlip = Rand_ZeroFloat(1.99f);
+        this->removeStateYaw = Rand_CenteredFloat(8000.0f) + -10000.0f;
+        this->cluckTimer = 30;
+        this->movementTimer = 100;
+        this->actionFunc = EnSyatekiNiw_Remove;
     }
 }
 
-void func_80B129EC(EnSyatekiNiw* this, PlayState* play) {
+void EnSyatekiNiw_Remove(EnSyatekiNiw* this, PlayState* play) {
     s32 pad;
     f32 phi_f2;
```

Comment:
```suggestion
    f32 rotYTargetOffset ;
```
Seems similar in function to the one above.